static/app/views/issueList/tagFilter.tsx

```diff
@@ -167,24 +172,38 @@ class IssueListTagFilter extends React.Component<Props, State> {
         {!tag.isInput && (
           <SelectControl
             clearable
+            aria-label={tag.key}
             placeholder="--"
             value={this.state.value}
             onChange={this.handleChangeSelect}
-            isLoading={this.state.isLoading}
+            isLoading={isLoading}
             onInputChange={this.handleChangeSelectInput}
             onFocus={this.handleOpenMenu}
-            noResultsText={
-              this.state.isLoading ? t('Loading\u2026') : t('No results found')
-            }
+            noResultsText={isLoading ? t('Loading\u2026') : t('No results found')}
             options={
               tag.predefined
                 ? tag.values &&
                   tag.values.map(value => ({
                     value,
                     label: value,
                   }))
-                : this.state.options
+                : options
             }
+            components={{
+              Option: ({
+                label,
+                ...optionProps
+              }: OptionProps<{
+                label: string;
+                value: string;
+              }>) => {
+                return (
+                  <components.Option label={label} {...optionProps}>
+                    <span data-test-id={`select-control-option-${label}`}>{label}</span>
```

Comment:
Seems like this should actual be done at our select component level instead of a one off here, no?

Reply:
I updated the code with another way to achieve this, maybe it looks better now